rj_gameplay/stp/utils/world_state_converter.py

```diff
@@ -243,7 +243,7 @@ def robot_creator(robot_state: RobotState, robot_status: RobotStatus) -> rc.Robo
 
 def worldstate_creator(partial_world_state: PartialWorldState,
                        robot_statuses: List[RobotStatus],
-                       game_info: rc.GameInfo, field: rc.Field,
+                       game_info: Optional[rc.GameInfo], field: rc.Field,
```

Comment:
```
Error: rj_gameplay/rj_gameplay/gameplay_node.py:168: error: Argument 3 to "worldstate_creator" has incompatible type "Optional[GameInfo]"; expected "GameInfo"
Error: rj_gameplay/rj_gameplay/gameplay_node.py:180: error: Argument 3 to "worldstate_creator" has incompatible type "Optional[GameInfo]"; expected "GameInfo"
```

Fixed the incompatible GameInfo errors in gameplay_node by making this Optional.